config/initializers/job_configurations.rb

```diff
@@ -206,3 +206,15 @@
   timeout: 300,
   callback: -> { Agreements::Reports::PartnerApiReport.new.call },
 )
+
+# Queue heartbeat job to DelayedJob
+JobRunner::Runner.add_config JobRunner::JobConfiguration.new(
+  name: 'Job Queue Heartbeat',
+  interval: 5 * 60, # 5 minutes
+  timeout: 4 * 60,
+  callback: -> do
+    HeartbeatJob.perform_later if IdentityConfig.store.ruby_workers_enabled
```

Comment:
Would it make sense to apply this condition at the `add_config` call? i.e. avoid adding the job at all if Ruby workers aren't enabled? Or do we want all configs added in all cases?

Reply:
+1 seems weird to me to have an empty job body, in order to pick up a config change we'd need a restart anyways?

Reply:
yep, thank you. I was initially unsure if config would be loaded at that point and never re-visited, but it is safe to do it before the job is added.